packages/blade/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -21,7 +21,7 @@ import { Counter } from '~components/Counter';
 import { Box } from '~components/Box';
 import { Card, CardBody, CardHeader, CardHeaderLeading } from '~components/Card';
 import { Alert } from '~components/Alert';
-import { Switch } from '~components/Switch';
+// import { Switch } from '~components/Switch';
```

Comment:
.

Reply:
Switch hasn't been rebranded can't use it yet.